easybuild/easyblocks/i/impi.py

```diff
@@ -210,7 +211,23 @@ def sanity_check_step(self):
             'dirs': [],
         }
 
-        super(EB_impi, self).sanity_check_step(custom_paths=custom_paths)
+        # Add minimal test program to sanity checks
+        try:
+            fake_mod_data = self.load_fake_module()
+        except EasyBuildError as err:
+            self.log.debug("Loading fake module failed: %s" % err)
+
+        impi_testsrc = os.path.join(self.installdir, 'test/test.c')
+        impi_testexe = os.path.join(self.builddir, 'mpi_test')
+        self.log.info("Building minimal MPI test program: %s", impi_testsrc)
+        build_test = "mpiicc %s -o %s" % (impi_testsrc, impi_testexe)
+        run_cmd(build_test, log_all=True, simple=True)
```

Comment:
@lexming Why not make the compilation command part of the sanity check command?

That way, you can avoid having to load the (fake) module above?

Reply:
good point, fixed :+1: